board/app/helpers/html_helper.php

```diff
@@ -1,7 +1,13 @@
 <?php
-
 function eh($string)
 {
     if (!isset($string)) return;
     echo htmlspecialchars($string, ENT_QUOTES);
 }
+
+function redirect($url)
```

Comment:
You may remove this helper function since you can use the function for redirection from the router.php :bow: